vision_msgs/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
I believe the root license is sufficient when matched with the package.xml declarations of Apache 2.0